src/features/session_replay/aggregate/index.js

```diff
@@ -304,6 +326,7 @@ export class Aggregate extends AggregateBase {
   /** Abort the feature, once aborted it will not resume */
   abort () {
```

Comment:
[no action required]

There's actually quite a couple `abort` places in here, so I'll add an abortcontroller abort here in shared-channel on my next Trace PR and listen to that instead of just SESSION_RESET

Reply:
good call